### ISSUE TITLE
Linkedin

### ISSUE THREAD

**Issue by Minaxi11** [anythingcodes/highlight-share#9]
> LinkedIn did not share a highlighted text

**Comment by scotteuser** [comment]
Yes, it seems that LinkedIn has switched to Open Graph so the option to share via LinkedIn should be removed
https://www.linkedin.com/help/linkedin/answer/46687/making-your-website-shareable-on-linkedin?lang=en
